CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.43.6] - 2023-07-10
+
```

Comment:
please add a commit message here to describe the change

Reply:
updated